packages/cli/config/jest.js

```diff
@@ -194,9 +194,16 @@ async function getProjectConfig(targetPath, extraConfig) {
     ...getRoleConfig(closestPkgJson?.backstage?.role),
   };
 
+  options.setupFilesAfterEnv = [];
+
+  if (options.testEnvironment === require.resolve('jest-environment-jsdom')) {
+    // FIXME https://github.com/jsdom/jsdom/issues/1724
+    options.setupFilesAfterEnv.push(require.resolve('cross-fetch/polyfill'));
```

Comment:
Feeling we want to make sure this is always first right? so `.unshift` given the change above